src/pytorch_lightning/strategies/launchers/multiprocessing.py

```diff
@@ -153,7 +153,7 @@ def _recover_results_in_main_process(self, worker_output: "_WorkerOutput", train
         # load last weights
         if worker_output.weights_path is not None:
             ckpt = self._strategy.checkpoint_io.load_checkpoint(worker_output.weights_path)
```

Comment:
Can you add a small comment explaining the "why" of having strict false hardcoded here?

Reply:
was this added? :chipmunk:

Reply:
added the requested comment